scheduler/src/cook/mesos.clj

```diff
@@ -207,6 +207,7 @@
                           (reify LeaderSelectorListener
                             (takeLeadership [_ client]
                               (log/warn "Taking mesos leadership")
+                              (swap! mesos-leadership-atom (constantly true))
```

Comment:
I think you want just `true` here. I expect this is left over from a refactor where you changed mesos-leadership-atom to hold a value instead of a function